src/datreant/core/trees.py

```diff
@@ -213,7 +217,7 @@ def __getitem__(self, path):
         def filt(path):
             fullpath = os.path.abspath(os.path.join(self.abspath, path))
 
-            if (os.path.isdir(fullpath) or path.endswith(os.sep) or
+            if (os.path.isdir(fullpath) or path.endswith("/") or
```

Comment:
The test for tree and treant are posix style, so path.endswith(os.sep) fails on win. 

Design question...should the "dtr[a/path/on/system]..." stay posix-style?